feat(kmp): update getting started and add manual setup page (#11497)
getsentry/sentry-docs: docs/platforms/kotlin-multiplatform/features/index.mdx

---
title: Features
sidebar_order: 1
description: "Learn about the features of Sentry's Kotlin Multiplatform SDK."
---

> Sentry's Kotlin Multiplatform SDK enables automatic reporting of errors and exception.
> The SDK provides a unified API for all supported platforms, directly leveraging the native SDKs for each platform.
> Therefore it also supports the same features as the native SDKs.
> 
> - [Apple features](/platforms/apple/features)
> - [Android features](/platforms/android/features)
> - [Java features](/platforms/java/features)

Reviewer: ```suggestion
Sentry's Kotlin Multiplatform SDK enables automatic reporting of errors and exceptions.
Because the SDK provides a unified API for all supported platforms directly leveraging the native SDKs for each platform, it also supports the same features as the native SDKs. Click on a link below to see relevant features for your platform:

- [Apple features](/platforms/apple/features)
- [Android features](/platforms/android/features)
- [Java features](/platforms/java/features)
```

> <Note>
> Some functionality might not yet be available in the shared Kotlin Multiplatform layer therefore it might be necessary to use the native SDKs directly or write your own platform specific code.
> 
> If you require access to the native SDK init options we recommend using the [Native Platform Options initialization](/platforms/kotlin-multiplatform/initialization-strategies/#native-platform-options).
> </Note>

Reviewer: ```suggestion
<Note>
Some functionality may not be available in the shared Kotlin Multiplatform layer yet. You may have to use the native SDKs directly, or write your own platform-specific code.

To access the native SDK init options, use the [Native Platform Options initialization](/platforms/kotlin-multiplatform/initialization-strategies/#native-platform-options).
</Note>
```
